### PR TITLE
[ci] Add rspec test for revision handling in PackageController#show

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

**Comment by ChrisBr** [comment]
LGTM

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -79,7 +79,7 @@ Metrics/AbcSize:
 # Offense count: 194
 # Configuration parameters: CountComments, ExcludedMethods.
 Metrics/BlockLength:
-  Max: 925
+  Max: 931
```
Comment (hennevogel) [inline .rubocop_todo.yml]: Looks like we should start splitting the package controller spec...

Comment (bgeuken) [inline .rubocop_todo.yml]: eventually yes^^

**Comment by mdeniz** [comment]
Travis is not passing.. looks like is because you need the VCR cassettes...

**Comment by hennevogel** [comment]
Superseded by #3115

**Pull request closed without merging**